src/main/java/edu/ufpr/cluster/algorithms/functions/InitializiationFunction.java

```diff
@@ -1,28 +0,0 @@
-package edu.ufpr.cluster.algorithms.functions;
-
```

Comment:
Please check your branch changes.
Does not makes muche sense remove this class.